keyboards/modelh/readme.md

```diff
@@ -0,0 +1,19 @@
+# Model H - A USB capable PCB for the IBM Model M keyboard
+
+![modelh](https://modelh.club/modelh_assembled.jpg)
```

Comment:
Could you rehost this on imgur or the like?

Reply:
Sure, will do.